### PR TITLE
Bridge Mitsubishi AC devices to Homebridge/Homekit locally

### PR DESCRIPTION
**Pull request by grzegorz914**
## Description:

* code structure cleanup.
* added Power, SetTemperature, Mode, SwingV, SwingH, FanSpeed to the JSON Object StatusSNS.MiElHVAC
This changes will be used with upcoming update for [homebridge-tasmota-control](https://github.com/grzegorz914/homebridge-tasmota-control) plugin which will be fully supported Mitsubishi AC devices locally and bridged it to the HomeKit.

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ ] The code change is tested and works with Tasmota core ESP8266 V.2.7.8
  - [x] The code change is tested and works with Tasmota core ESP32 V.3.1.0.240926
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by grzegorz914** [comment]
@arendst
it is ready for review

Thanks
Grzegorz

**Comment by Jason2866** [comment]
@grzegorz914 Plese undo format & style changes

**Comment by grzegorz914** [comment]
@Jason2866  ready

**Comment by grzegorz914** [comment]
@arendst @Jason2866 
please review and merge this if You have time because I have other PR and add AirDirection control (even, direct, indirect) for devices with ISEE sensor.

File: tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino [hunk 1]
```diff
@@ -1144,42 +1193,34 @@ miel_hvac_sensor(struct miel_hvac_softc *sc)
...
+		ResponseAppend_P(PSTR(",\"Operation\":\"%s\""),
+		    s->operation ? "ON" : "OFF");
+		ResponseAppend_P(PSTR(",\"Compressor\":\"%s\""),
+		    s->compressor ? "ON" : "OFF");	
 	}
 
 	if (sc->sc_temp.type != 0) {
-		ResponseAppend_P(PSTR("%s" "\"roomtemp\":\"%s\""), sep,
+		ResponseAppend_P(PSTR(",\"roomtemp\":\"%s\""),
 		    ToHex_P((uint8_t *)&sc->sc_temp, sizeof(sc->sc_temp),
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino:1212]: Use Uppercase first char pls like Roomtemp

File: tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino [hunk 2]
```diff
@@ -1144,42 +1193,34 @@ miel_hvac_sensor(struct miel_hvac_softc *sc)
...
 		    hex, sizeof(hex)));
-
-		sep = ",";
 	}
 
 	if (sc->sc_status.type != 0) {
-		ResponseAppend_P(PSTR("%s" "\"status\":\"%s\""), sep,
+		ResponseAppend_P(PSTR(",\"status\":\"%s\""),
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino:1217]: Ditto

File: tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino [hunk 3]
```diff
@@ -1144,42 +1193,34 @@ miel_hvac_sensor(struct miel_hvac_softc *sc)
...
 		    ToHex_P((uint8_t *)&sc->sc_status, sizeof(sc->sc_status),
 		    hex, sizeof(hex)));
-
-		sep = ",";
 	}
 
 	if (sc->sc_stage.type != 0) {
-		ResponseAppend_P(PSTR("%s" "\"stage\":\"%s\""), sep,
+		ResponseAppend_P(PSTR(",\"stage\":\"%s\""),
```
Comment (arendst) [inline tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino:1223]: Ditto

**Pull request merged**